css-view-transitions-2/Overview.bs

```diff
@@ -117,7 +117,7 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	1. Once it's time to [=unload=] the old document, if the navigation is [=same origin=]
 		and the old {{Document}} has opted in to cross-document view-transitions, the old state is captured.
 
-	1. An event named {{PageRevealEvent|reveal}} is fired on the new {{Document}}, with a `viewTransition` property,
+	1. An event named {{ReadyToRenderEvent|readytorender}} is fired on the new {{Document}}, with a `viewTransition` property,
```

Comment:
The discussion on this is still ongoing in https://github.com/whatwg/html/issues/9315 (as you know). I'm fine with the change, but we'll just change this again it feels like

Reply:
Yea that's OK.